nautobot/extras/templates/extras/object_new_contact.html

```diff
@@ -3,8 +3,10 @@
 {% load helpers %}
 
 {% block tabs %}
-    {% if not editing %}
+    {% if not editing and not map %}
         {% include 'extras/inc/object_contact_header.html' with active_tab='contact' %}
+    {% else %}
+        {% include 'extras/inc/map_contact_header.html' with active_tab='contact' %}
     {% endif %}
```

Comment:
Is this logic correct? I'd naively expect something more like:

```
if not editing:
    if not map:
        include object_contact_header
    else:
        include map_contact_header
```

Won't this as currently implemented result in the "map" header being added in the case of `editing`?